pkg/log/logger.go

```diff
@@ -0,0 +1,95 @@
+package log
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"log/slog"
+	"os"
+)
+
+var ctx context.Context
+
+const (
+	levelTrace = slog.Level(-8)
+	levelFatal = slog.Level(12)
+)
+
+var levelNames = map[slog.Leveler]string{
+	levelTrace: "TRACE",
+	levelFatal: "FATAL",
+}
+
+var LogFileName string
+var LogLevel int
+
+var logger *slog.Logger
+
+func InitLogger() error {
+
+	ctx = context.Background()
```

Comment:
I think we should accept a context from the args instead of creating a new one here.